02.calendar/cal.rb

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+require 'optparse'
+require 'date'
+
+options = ARGV.getopts('y:', 'm:')
+
+input_year = options["y"]
+input_month = options["m"]
+
+target_year = input_year.nil? ? Date.today.year : input_year.to_i
```

Comment:
```suggestion
year = options["y"] ? options["y"].to_i : Date.today.year
```

こんなふうにした方がちょっとコードがシンプルになりそうです